specification/security/resource-manager/Microsoft.Security/preview/2023-09-01-preview/securityConnectorsDevOps.json

```diff
@@ -0,0 +1,3785 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft.Security",
+    "description": "REST APIs for Defender for DevOps",
+    "version": "2023-09-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/securityConnectors/{securityConnectorName}/devops/default/listAvailableAzureDevOpsOrgs": {
+      "post": {
+        "tags": [
+          "default"
+        ],
+        "summary": "Returns a list of all Azure DevOps organizations accessible by the user token consumed by the connector.",
+        "operationId": "AzureDevOpsOrgs_ListAvailable",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
```

Comment:
Will v5 conflict with other endpoints in Microsoft.Security? We updated to v5 because of [this request](https://github.com/Azure/azure-rest-api-specs/pull/24384#discussion_r1234777022) on this review. If using v5 will cause AutoRest to fail because of usage of v2 elsewhere in Microsoft.Security, we need to communicate that as the reason for not following the guidance.

Reply:
v2 in my PR is v5 copied with alternate descriptions - a move to v5 will have to be coordinated in a new, cross MDC API version